test/evergreen.yml

```diff
@@ -2260,19 +2224,16 @@ buildvariants:
     - name: make-check-msan-test
     - name: compile-ubsan
     - name: ubsan-test
-    # Temporarily disabled
-    # - name: linux-directio
-    #   distros: ubuntu1804-build
+    - name: linux-directio
+      distros: ubuntu1804-build
     - name: syscall-linux
     - name: make-check-asan-test
     - name: configure-combinations
-    # Temporarily disabled
     # - name: checkpoint-filetypes-test
```

Comment:
How about these? I saw you uncommented the job above.

Reply:
The test is meant to be kept disabled. WT-6063 was created to address and the failure and re-enable it. I uncomment the task definition segment and left the task commented out at the build variant level, which in fact will achieve the same effect of disabling the task. The benefit is next time  when the test is ready to be enabled we only need 1-line change.